Configurateur/front/lib/screens/login/login.dart

```diff
@@ -26,175 +29,208 @@ class LoginScreen extends StatefulWidget {
 /// page de connexion pour le configurateur
 ///
 class LoginScreenState extends State<LoginScreen> {
+  String? token = '';
+  String? userMail = '';
+
+  void checkToken() async {
+    token = await storageService.readStorage('token');
+    storageService.getUserMail().then((value) => userMail = value);
+    setState(() {});
+  }
+
+  @override
+  void initState() {
+    super.initState();
+    checkToken();
+  }
+
   @override
   Widget build(BuildContext context) {
     final GlobalKey<FormState> formKey = GlobalKey<FormState>();
     String mail = '';
     String password = '';
     dynamic response;
-    Map<String, dynamic> decodedToken;
 
     return Scaffold(
-        appBar: CustomAppBar(
-          'Connexion',
-          context: context,
-        ),
-        body: Center(
-            child: FractionallySizedBox(
-                widthFactor: 0.2,
-                heightFactor: 0.7,
-                child: Form(
-                    key: formKey,
-                    child: Column(children: <Widget>[
-                      const SizedBox(height: 10),
-                      TextFormField(
-                        key: const Key('email'),
-                        decoration: InputDecoration(
-                          hintText: 'Entrez votre email',
-                          labelText: 'Adresse e-mail',
-                          border: OutlineInputBorder(
-                            borderRadius: BorderRadius.circular(30.0),
+        body: FooterView(
+            footer: Footer(
+              child: CustomFooter(context: context),
+            ),
+            children: [
+          LandingAppBar(context: context),
```

Comment:
why is it indented like that ?

Reply:
good question

Reply:
Fixed, problem with ) or ] with missing comma